src/libs/actions/Policy/Tag.ts

```diff
@@ -618,6 +618,7 @@ function setPolicyTagsRequired(policyID: string, requiresTag: boolean, tagListIn
                 value: {
                     [policyTag.name]: {
                         required: policyTag.required,
+                        errors: ErrorUtils.getMicroSecondOnyxError('workspace.tags.genericFailureMessage'),
```

Comment:
Use errorFields because we have more than one field that could error out. If we use errors and something goes wrong the error message will appear in many places (e.g. tag name and tag required)

<img width="383" alt="Screenshot 2024-06-03 at 9 47 07 PM" src="https://github.com/Expensify/App/assets/16493223/fac7fab9-7145-4f94-bee0-ba3a8ae9b554">


```js
Onyx.merge('policyTags_CE194D2E3EC60BA3', {"Country": {errors: {0: "error here"}}})
```

Reply:
I hope I did a correct thing here 😅 

https://github.com/Expensify/App/pull/42972/commits/7e5a4dbe5182e7f9c046ea313a12ff9630576cf3